setup.cfg

```
@@ -1,19 +1,19 @@
 [metadata]
 name = multicast
-version = 2.0.4
+version = 2.0.8-alpha
```

Remember to update the version before the release.
```suggestion
version = 2.0.8
```